VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+Where a member of the team is employed by a downstream stakeholder, the member does not give their employer prior notice of any vulnerabilities.
+In order to reduce the risk of accidental disclosure of vulnerability in the early stages, membership of this team is intentionally limited to a small number of people.
+
+# Acronyms
+
+To aid reading this document it helps to understand some key acronyms:
+
+| Acronym  | Description                                                     |
+| -------- | --------------------------------------------------------------- |
+| CNA      | [CVE Numbering Authority](https://cve.mitre.org/cve/cna.html)   | 
+| CVE      | [Common Vulnerabilities and Exposures](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures) |
+| KCSA     | Kata Containers Security Announcement                           |
+| KCSN     | Kata Containers Security Note                                   |
+| KCST     | Kata Containers Security Team                                   | 
+| PR       | [A GitHub Pull Request](https://help.github.com/articles/about-pull-requests/) |
+| VMT      | The Kata Containers Vulnerability Management Team               |
+
+# Supported versions
+
+> **FIXME** Before we can make statements about which stable versions we fix, we need to define the Kata Containers release and maintenance process.
+
+The VMT coordinates patches fixing vulnerabilities in supported stable branches of Kata Containers, in addition to the master branch (next version under development).
+
+# Process
+
+Each security bug is assigned a VMT *coordinator* (a member from the VMT) that will drive the fixing and disclosure process.
+Here are the steps we follow.
+
+![VMT process workflow](VMT-process.png)
+
+## Reception
+
+A report is received via an email to the [VMT mailing list](mailto:kata-vmt@lists.katacontainers.io).
+
+> **FIXME** We need to decide where and how reports are handled.  The obvious choice is to use a private GitHub repo alongside the other Kata Containers repos - but, we would have to pay for that (private repos are not free).  Another alternative may be to use any OSF infrastructure already in place and in use for other OSF projects, if any exists.
+> This document will be drafted under the assumption we are going to use private GitHub repos and issues, and updated later if that is not the case.
+
+The first steps performed by the VMT are to confirm the validity of the report, create a private GitHub Issue if necessary, prefix the description with an [embargo reminder](templates/reception-embargo-reminder.md), create a Kata Containers Security Announcement ([KCSA](templates/KCSA.md)) Issue, and subscribe the project's core security review team for confirmation of impact and determination of affected branches.
```

Comment:
Could we make this a bullet list? It would add hugely to the clarity of what needs to be done imho. In fact, why not make it a checklist as we have for the release checklist:

- https://github.com/kata-containers/documentation/blob/master/Release-Checklist.md